Demo/Ser_Des/ser_des.h

```diff
@@ -0,0 +1,104 @@
+#ifndef SER_DES_H
+#define SER_DES_H
+
+#include <iostream>
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/mman.h>
+#include <cstring>
+#include <cerrno>
+
+#include <type_traits>
+
+struct file {
+    int fd;
+
+    explicit file(const std::string& path)
+        : fd(open(path.c_str() , O_RDWR | O_CREAT , 0666)){}
+
+    void write_to_file(const int number)const {
+        write(fd , &number , sizeof(number));
+    }
+
+    void read_from_file(int& number)const {
+        lseek(fd , 0 , SEEK_SET);
+        read(fd , &number, sizeof(number));
+    }
+
+    ~file() {
+        close(fd);
+    }
+};
+
+constexpr size_t block_size = 4096;
+constexpr int magic_number = 564982;
```

Comment:
Make the number hex. 0x89EF6
Rename the variable to something like "serializationMagic"